gnat2goto/driver/tree_walk.adb

```diff
@@ -1130,6 +1138,130 @@ package body Tree_Walk is
       end if;
    end Do_Full_Type_Declaration;
 
+   ---------------------
+   -- Name_Has_Prefix --
+   ---------------------
+
+   function Name_Has_Prefix (N : Node_Id; Prefix : String) return Boolean
```

Comment:
I agree, this is again one of those cases where it would be more natural to implement at the cbmc side, especially because the range checks are already inserted there. But also a bit trickier, I suppose. 
That is just a temporary solution (now that I've said this, chances are this will stick around for a while). On the plus side, the symbol table is a bit smaller this way.

The other changes related to null procedures and RTS calls, however, should still be merged.